Enable to install

### Description

I'm not able to install dependencies.

### Steps to Reproduce

1. Clone the repository
2. Run `npm install`

**Expected behavior:**

All depencendies must be installed.

**Actual behavior:**

```sh
$ npm install
npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
npm WARN deprecated browserslist@2.11.3: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
npm WARN deprecated swig@1.4.0: This package is no longer maintained
npm WARN deprecated graceful-fs@3.0.11: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm WARN deprecated node-uuid@1.4.0: Use uuid module instead
npm WARN deprecated transformation-matrix-js@2.7.6: Package no longer supported. Contact support@npmjs.com for more info.
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm ERR! code 1
npm ERR! Command failed: /usr/bin/git checkout 4.0
npm ERR! error: pathspec '4.0' did not match any file(s) known to git.
npm ERR! 
```

**Reproduces how often:**

All the time.

--- a/package.json
+++ b/package.json
@@ -35,7 +35,7 @@
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-react": "^7.6.1",
     "glob": "^7.1.2",
-    "gulp": "github:gulpjs/gulp#4.0",
+    "gulp": "^4.0.0",
     "gulp-babel": "^7.0.1",
     "gulp-cached": "^1.1.1",
     "gulp-concat": "^2.6.1",

